todo_app/trello.py

```diff
@@ -0,0 +1,81 @@
+from types import MethodDescriptorType
+from dotenv import load_dotenv
+import requests
+import itertools
+import json
+import os 
+
+
+class Trello:
+    def __init__(self):
+        load_dotenv()
+        self.key = os.environ.get('TRELLO_KEY')
+        self.token = os.environ.get('TRELLO_TOKEN')
+        self.url = os.environ.get('TRELLO_BASE_URL')
+        self.board = os.environ.get('TRELLO_BOARD_ID')
+        self.username = os.environ.get('TRELLO_USER')
+        self.list_ids = self.get_card_list_ids()
+
+    def make_api_call_and_check_response(self, url, method='get', data=None):
+        complete_url = f"{self.url}" + url + f"?key={self.key}&token={self.token}"
+        resp = eval("requests." + method.lower() + "('" + complete_url + "', data=" + str(data) + ")")
```

Comment:
Rather than do something risky like this, you could just use:
```suggestion
        resp = requests.request(method, complete_url, data=data)
```

You can normally give popular libraries the benefit of the doubt and check if they do what you want.

Also, note that you've used `data` which sets the request body. If you look at the Trello documentation, they actually ask for things like name/idList to be query parameters. But it seems they've generously made the API accept either.